app/views/providers/partners/address_lookups/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= form_with(
+      model: @form,
+      scope: :address_lookup,
+      url: providers_legal_aid_application_partners_address_lookup_path(@legal_aid_application),
+      method: :patch,
+      local: true,
+    ) do |form| %>
+
+  <%= page_template(
+        page_title: t(".page_title"),
+        template: :basic,
+        form:,
+      ) do %>
+
+    <%= form.govuk_fieldset legend: { size: "xl", tag: "h1", text: t(".heading") }  do %>
+      <div class="govuk-!-padding-2"></div>
```

Comment:
Best practice: the relevant spacing should already be applied by the design system, if additional spacing is required, best to add padding to the fieldset or margin to the text field.

Reply:
These were duplicated from the original views and I'm not sure why these were added in the first place!
![image](https://user-images.githubusercontent.com/6757677/229800913-9ead2ab5-474c-4a3b-892e-5eeca54cfd25.png)
If I remove the padding entirely...
![image](https://user-images.githubusercontent.com/6757677/229801092-6b0e1adf-afb3-4d5a-aa87-28ea4df297bc.png)
I'm not sure I see a need for this, do you @naseberry?